code/code/misc/damage.cc

```diff
@@ -1124,7 +1130,17 @@ int TBeing::damageEpilog(TBeing* v, spellNumT dmg_type) {
             if (f->follower->desc) {
               f->follower->desc->session.groupKills++;
               f->follower->desc->career.group_kills++;
-              grouplist = format("%s, %s") % grouplist % f->follower->getName();
+            }
+              
+            // discord group naming
+            if (f->follower->getName() != getName()) {
```

Comment:
What exactly does this line do? I can't quite wrap my head around these pointers. Is it double-checking that the follower is following the leader? Checking that the leader is not following itself?

Reply:
see above - the current entity will be either the master or in the follower list, so we check to make sure they don't get added again.